examples/AzureSDKDemoSwift/Source/Common.swift

```diff
@@ -99,7 +99,8 @@ struct AppState {
             )
             let options = StorageBlobClientOptions(
                 apiVersion: StorageBlobClient.ApiVersion.latest.rawValue,
-                logger: ClientLoggers.none
+                logger: ClientLoggers.none,
+                maxConcurrency: 2
```

Comment:
The lower this is, the less we experience concurrency issues. Once we have those fixed, my preference would be for the default to simply be OS determined.